instrumentation/spring/spring-rmi/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/springrmi/server/ServerSingletons.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.springrmi.server;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.api.instrumenter.SpanKindExtractor;
+import io.opentelemetry.instrumentation.api.instrumenter.rpc.RpcSpanNameExtractor;
+import io.opentelemetry.instrumentation.api.util.ClassAndMethod;
+
+public final class ServerSingletons {
+
+  private static final Instrumenter<ClassAndMethod, Void> INSTRUMENTER;
+
+  static {
+    ServerAttributesExtractor attributesExtractor = new ServerAttributesExtractor();
+
+    INSTRUMENTER =
+        Instrumenter.<ClassAndMethod, Void>builder(
+                GlobalOpenTelemetry.get(),
+                "io.opentelemetry.springrmi",
```

Comment:
to match the module name
```suggestion
                "io.opentelemetry.spring-rmi-5.0",
```